### PR TITLE
Create fileRepository schema

### PR DESCRIPTION
**Pull request by skoehnen**
Implementation of #79

### CODE REVIEW

File: v3.0/products/fileRepository.schema.json [hunk 1]
```diff
@@ -1 +1,51 @@
...
+    "@id": {
+      "type": "string",
+      "description": "Metadata node identifier."
+    },
+    "hash": {
+      "description": "Hash generated for this file repository.",
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/HASH"
+    },
+    "mediaType": {
+      "description": "Type of media stored in this repository.",
```
Comment (lzehl) [inline v3.0/products/fileRepository.schema.json]: "Media type (mime type) of this repository."

File: v3.0/products/fileRepository.schema.json [hunk 2]
```diff
@@ -1 +1,51 @@
...
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/MEDIA_TYPE"
+    },
+    "name": {
+      "description": "Name of this repository.",
+      "type": "string"
+    },
+    "hostedBy": {
+      "description": "Hosting provider for this repository.",
```
Comment (lzehl) [inline v3.0/products/fileRepository.schema.json:38]: "Hosting institution/organization for this repository"

Comment (lzehl) [reply v3.0/products/fileRepository.schema.json:38]: I originally thought that this is a controlledTerm with values like "EBRAINS", "GitHub" etc. but it actually should reference to an organization which all of these are. I'm sorry that I just realized this now

File: v3.0/products/fileRepository.schema.json [hunk 3]
```diff
@@ -1 +1,51 @@
...
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/CONTROLLED_TERM"
```
Comment (lzehl) [inline v3.0/products/fileRepository.schema.json]: change to ref to organization.schema.json

**Review by lzehl — CHANGES_REQUESTED** [review]

File: v3.0/products/fileRepository.schema.json (hunk 3 above)
Comment (skoehnen) [reply v3.0/products/fileRepository.schema.json]: Fixed 👍

File: v3.0/products/fileRepository.schema.json (hunk 2 above)
Comment (skoehnen) [reply v3.0/products/fileRepository.schema.json:38]: Okay, this is handled by switching to organization.

File: v3.0/products/fileRepository.schema.json (hunk 1 above)
Comment (skoehnen) [reply v3.0/products/fileRepository.schema.json]: Fixed 👍

**Comment by skoehnen** [comment]
All changes should be implemented.

**Review by lzehl — APPROVED** [review]

**Comment by lzehl** [comment]
thanks @skoehnen

**Pull request merged**